utils/prometheus-node-exporter-lua/files/usr/lib/lua/prometheus-collectors/dawn.lua

```diff
@@ -0,0 +1,49 @@
+local ubus = require "ubus"
+
+local function scrape()
+  local metric_dawn_ap_channel_utilization = metric("dawn_ap_channel_utilization","gauge")
+  local metric_dawn_ap_station_count = metric("dawn_ap_station_count","gauge")
+  local metric_dawn_station_signal = metric("dawn_station_signal","gauge")
+
+  local u = ubus.connect()
+  local network = u:call("dawn", "get_network", {})
+
+  for ssid, ssid_table in pairs(network) do
+    for ap, ap_table in pairs(ssid_table) do
+
+      local ht_support = (ap_table['ht_support'] == true) and 1 or 0
+      local vht_support = (ap_table['vht_support'] == true) and 1 or 0
+
+      local labels = {
+        ssid = ssid,
+        bssid = ap,
+        freq = ap_table['freq'],
+        ht_support = ht_support,
+        vht_support = vht_support,
+        neighbor_report = ap_table['neighbor_report'],
```

Comment:
What kind of value is this? Labels should be simple strings, nothing more. The neighbor reports should probably be a separate metric.

Reply:
Neighborreport is just a statis string that won't change. 802.11v is based on it. Maybe I will remove it?

Reply:
Oh, so it's more like a boolean, whether the client submits them or is capable of submitting them?

Reply:
Actually there is information included that allows a client to immidiately roam to some AP (BSSID, Channel, ...)

Reply:
I guess that can change alot when a client is moving, and labels are not supposted to be short-lived.

Maybe expose those as separate metrics `dawn_station_neighbor_report_dbm` with appropriate labels?

Reply:
They don't change. That is static information that APs exchange and give the client on request. :)
Like "Hey AP I want to roam, but I do not want to perform a scan, please give me roaming canidates".
Or what I do: "Hey Client here is a list of APs (as neighborreport) that you should immidiately connect to" :)